corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1000,6 +1004,10 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
             }
         },
 
+        scrollContainer: function () {
```

Comment:
Since it's just returning a jQuery selector for a constant, could this be a property instead of a function? I don't remember how this is handled in Marionette-- maybe it _has to_ be a function. It might also be nice to call it `$scrollContainer` to indicate it's already a jQuery selector for an element. Not blocking.